setup-yarn-2.sh

```diff
@@ -62,3 +63,4 @@ sudo -u hduser sh -c '/usr/local/hadoop/sbin/mr-jobhistory-daemon.sh start histo
 
 # Check status
 sudo -u hduser -u hduser jps
+
```

Comment:
Why are we modifying this file? seems irrelevant to the dashboard.  If it needs fixed should it be a separate PR?